lib/db_utils.pm

```diff
@@ -232,3 +232,13 @@ sub check_postgres_db {
     }
     return 1;
 }
+
+is_ok_url {
+    # url connectivity check
+    my ($url) = @_;
+    return unless ($url and script_run('curl --help>/dev/null') == 0);
+
+    $args{timeout} //= 90;
+    return 0 if (script_run('curl -skf ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0) != 0);
+    return 1;
```

Comment:
These 2 lines can be merged like this:

```suggestion
    return (script_run('curl -skf ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0) == 0);
```

Reply:
ok, after performed local unit-checks, I agree to make it simpler and delegate all the checks to the script_run result.